contracts/reserve-auction/src/error.rs

```diff
@@ -10,36 +11,36 @@ pub enum ContractError {
     #[error("{0}")]
     PaymentError(#[from] PaymentError),
 
+    #[error("{0}")]
+    MarketplaceCommonError(#[from] MarketplaceCommonError),
```

Comment:
Maybe just `MarketplaceError`?

Reply:
Also maybe common -> std?

Reply:
Ok I've changed it to `MarketplaceStdError`